package.json

```diff
@@ -103,6 +103,9 @@
   "engines": {
     "node": "^6.x.x || ^7.x.x"
   },
+  "jest": {
+    "notify": true
```

Comment:
Somehow this ended up here, still very good feature, see:

![image](https://cloud.githubusercontent.com/assets/123822/22351644/2d260e36-e419-11e6-9df1-46e98dac19df.png)